doc/onboarding.md

```diff
@@ -35,51 +36,99 @@ Boulder Experimental Mathematics Lab.)
     https://code.visualstudio.com/docs/editor/extension-marketplace.
 
 2. If you don't have [Git](https://git-scm.com/) installed, install it. We use
-   Git to keep track of the different versions of our code.
+   Git to keep track of the different versions of our code. _Mac_: The easiest
+   way to install git is to install the Mac "developer tools." You can do so
+   by opening Settings > General > Software Update and then (in a terminal, as
+   mentioned above) executing `xcode-select --install`. Then follow the
+   prompts in the Software Update window; your computer may need to restart
+   for the changes to take effect.
 3. If you don't have [NodeJS](https://nodejs.org/en/) installed, install it.
    NodeJS allows you to run JavaScript outside of a web browser. We use it in
-   our front end code base.
+   our front end code base. _Mac_: You can obtain Node by first installing the
+   "node version manager" via:
+
+    ```
+    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.2/install.sh | bash
+    ```
+
+    [Note that you may need to change the `v0.40.2` portion of the above URL
+    to the latest version of nvm; it was current at the time of this writing,
+    but you can always find the latest release on its
+    [GitHub page](https://github.com/nvm-sh/nvm/releases).] Now either log out
+    and log back in for the nvm installtion to take effect, or instead just
+    execute `. ~/.nvm/nvm.sh`. Finally to install Node, choose the version of
+    Node you want (latest was 23 at the time of this writing, but you can
+    always look up the
+    [available releases](https://nodejs.org/en/about/previous-releases). Then
+    execute the command `nvm install [DESIRED_VERSION]`, e.g.,
+    `nvm install 23`.
+
 4. Make sure you have [Make](https://linuxhandbook.com/using-make/) installed.
-   There's a guide for Linux on that page; or you can try these other links
-   for
-   [Windows](https://stackoverflow.com/questions/32127524/how-to-install-and-use-make-in-windows)
-   or [MacOS](https://stackoverflow.com/questions/1469994/using-make-on-os-x).
-5. If you don't have [Python 3](https://www.python.org/) installed, install
-   it. We use Python in frontscope for our documentation site.
-6. Make sure you have a `venv` module by running the following command:
-    ```sh
-    python3 -m venv -h
+   There's a guide for Linux on that page; or you can try this link for
+   [Windows](https://stackoverflow.com/questions/32127524/how-to-install-and-use-make-in-windows).
+   _Mac_: Unfortunately, the version of make that Apple includes with its
```

Comment:
`make` (for readability)?  I don't know if it's consistent with our other docs, it's just grammatically confusing in this case because "make" is a regular english word.